PATCH-P1-FIX-SUSE: set libdir for ppc64 correctly

The patch sets correct libdir for ppc64

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: mysql-5.5.21/cmake/install_layout.cmake
===================================================================
--- mysql-5.5.21.orig/cmake/install_layout.cmake
+++ mysql-5.5.21/cmake/install_layout.cmake
@@ -138,7 +138,7 @@ SET(INSTALL_BINDIR_RPM
 SET(INSTALL_SBINDIR_RPM                 "sbin")
 SET(INSTALL_SCRIPTDIR_RPM               "bin")
 #
-IF(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64")
+IF(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64" OR CMAKE_SYSTEM_PROCESSOR MATCHES "ppc64" )
   SET(INSTALL_LIBDIR_RPM                "lib64")
   SET(INSTALL_PLUGINDIR_RPM             "lib64/mysql/plugin")
 ELSE()
